qcfractal/tests/test_collections.py

```diff
@@ -1151,6 +1151,12 @@ def test_reactiondataset_dftd3_dataset_plaintextview_write(reactiondataset_dftd3
     ds.to_file(tmpdir / "test.tar.gz", "plaintext")
 
 
+@pytest.mark.slow
```

Comment:
This seems fast?

Reply:
I'm marking everything that requires a connection to QCA as slow. It might make sense to have a different mark, but I think QCA-dependent tests should not run by default.